packages/compass-connections/src/components/connection-list/connection-list.tsx

```diff
@@ -166,7 +166,7 @@ function ConnectionList({
 
   const { theme } = useTheme();
 
-  const useNewSidebar = process?.env?.COMPASS_SHOW_NEW_SIDEBAR === 'true';
+  const useNewSidebar = process?.env?.COMPASS_SHOW_NEW_SIDEBAR !== 'false';
```

Comment:
I noticed that we did the same for query bar. Is it a new convention that we are following: keeping the old code, just flipping the flag?

Reply:
Don't know about conventions, just trying to keep the PRs small. This is a nice small unit that can easily be reverted. There's quite a lot of stuff to remove - because of all these checks it isn't just a matter of removing the legacy components folder. So I figured that's best done in its own step.

Reply:
Gotcha, makes sense, thanks for context!